### DISCUSSION TITLE
Error:  Error evaluating 'gcode_macro PRINT_START:gcode': jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'BED

### DISCUSSION THREAD

**Discussion by wendyr21** [bassamanator/Sovol-SV06-firmware#51] [General]
Hi, can someone help me. I didnt change any of the config files just copied them in. I think i dont have the start code setup correctly in Prusa. any guidance would be appreciated. tx

**Comment by bassamanator** [comment]
Please see `Adjust your slicer` section.
